lib/player.rb

```diff
@@ -1,7 +1,4 @@
 module TicTacToe
-  
-  MARKERS = ["X", "O"]
-
   class Player
 
```

Comment:
I think it's fine to leave this here as it's specific to the Player class, but I would put it below the Player class declaration so that it'd be referred to as `TicTacToe::Player::MARKERS` elsewhere in the code.